### DISCUSSION TITLE
Newbie Q

### DISCUSSION THREAD

**Discussion by alanrock24** [mworion/MountWizzard4#372] [Q&A]
I have just installed MW4 on Win11 and got connected to the mount. I use NINA, and have always just used the 10u plugin a to build models. It was always very easy. Just generate points and build, but MW4 is better and more helpful, but quite honestly I don't know where to start. I can't even figure out how to apply a horizon mask like I have done in NINA. 
Can anyone help me to get started?  I have looked in the manual and still cannot figure out how to do things.
Thanks!
Alan

**Comment by WillB-2024** [comment]
Hi Alan.

I faced the same challenge when starting out with MW4 a few years ago, If you haven't reviewed them yet have a look at Michael's MW4 video channel on >[YouTube](https://www.youtube.com/@orion49m/videos)<.

Many of these videos are quite old now and MW4 has evolved a bit since these were created but the basics are still there and the most relevant videos are quite short, even though I'm deaf and can't hear the narration (if any) I managed to pick up enough to get started.

If your 10u is in a fixed observatory the biggest hurdle I found is remembering how to use MW4 between sessions as I only change scopes and remodel every eighteen months or so, mobile users who remodel every session won't face that problem.

Will.
